website/docs/components/copy/snippet/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,55 @@
+## Usage
+
+### When to use
+
+- To allow users to copy code snippets, such as URLs, tokens, IDs, etc.
+
+### When not to use
+
+- To copy content outside its container, use [Copy Button](/components/copy/button).
+- In a code block, use [Copy Button](/components/copy/button).
+
+### Effective Usage
+
+- There’s no limit to the number of `Copy::Snippet` components on a page, but maintain a balance to avoid overwhelming the user or cluttering the interface.
+- Use the `Copy::Snippet` judiciously for crucial information that users frequently need. This keeps the copied content relevant and valuable without causing confusion or clutter.
+
+!!! Dont
+
+Take caution when using for full code snippets, instead consider using a more prominent code block with a [Copy Button](/components/copy/button).
+
+![Example of the copy snippet component not using proper usage] (/assets/components/copy/copy-snippet-dont.png)
+!!!
+
+## Color
+The `Copy::Snippet` comes in two colors: `primary` and `secondary`
```

Comment:
We do often indicate property values within the guidelines, so I think this is fine, but I would update the examples to use a more "real-world" value instead of "Primary" and "Secondary". @andgen404 is there another example of proper usage of the primary variant from the audit? Couldn't hurt to add that example in as a "do" too.